CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ All notable changes to this project will be documented in this file.
 * ### ALL
     * #### Added
         * Support for Python 3.10
+        * MeasurementLink support
```

Comment:
Let's be more precise and only put this line in those drivers that actually got the support added. I believe the list is:

✅ nidcpower
✅ nidigital
✅ nidmm
✅ nifgen
⛔️ nimodinst
✅ niscope
⛔️ nise
✅ niswitch
⛔️ nitclk

Reply:
did it